### PR TITLE
Correct Jenkinsfile to use the recommended `useContainerAgent` parameter

### PR DESCRIPTION
**Pull request by jmMeessen**
Replace the use of the `useAci` by the recommended `useContainerAgent` (as recommended in https://issues.jenkins.io/browse/INFRA-2918.)

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+tasks:
```
Comment (jglick) [inline .gitpod.yml:1]: keep it to #22

File: .mvn/extensions.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<extensions xmlns="http://maven.apache.org/EXTENSIONS/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/EXTENSIONS/1.0.0 http://maven.apache.org/xsd/core-extensions-1.0.0.xsd">
```
Comment (jglick) [inline .mvn/extensions.xml:1]: keep it somewhere else

File: .mvn/maven.config [hunk 3]
```diff
@@ -0,0 +1,3 @@
+-Pconsume-incrementals
```
Comment (jglick) [inline .mvn/maven.config:1]: ditto

File: Jenkinsfile [hunk 4]
```diff
@@ -1,6 +1,6 @@
     
 buildPlugin(
-    useAci: true,
+    useContainerAgent: true,
```
Comment (jglick) [inline Jenkinsfile:3]: :+1: for this line

File: .gitpod.yml (hunk 1 above)
Comment (jmMeessen) [reply .gitpod.yml:1]: Fixed by merging #22

**Pull request merged**